odl/util/utility.py

```diff
@@ -49,7 +49,12 @@ def array1d_repr(array, nprint=6):
     """
     assert int(nprint) > 0
 
-    if len(array) <= nprint:
+    if array.ndim == 0:
+        if array.size == 1:
+            return repr(np.array(array, ndmin=1)[0])
+        else:
+            return ''
```

Comment:
Is this truly the right result in this case? Shouldn't it be `np.array()` or something like that?